sdk/keyvault/Azure.Security.KeyVault.Shared/src/ChallengeBasedAuthenticationPolicy.cs

```diff
@@ -21,7 +21,7 @@ internal class ChallengeBasedAuthenticationPolicy : BearerTokenAuthenticationPol
         private static readonly ConcurrentDictionary<string, ChallengeParameters> s_challengeCache = new();
         private ChallengeParameters _challenge;
 
-        public ChallengeBasedAuthenticationPolicy(TokenCredential credential, bool disableChallengeResourceVerification) : base(credential, Array.Empty<string>())
+        public ChallengeBasedAuthenticationPolicy(TokenCredential credential, bool disableChallengeResourceVerification, bool enableCAE = false) : base(credential, Array.Empty<string>())
```

Comment:
```suggestion
        public ChallengeBasedAuthenticationPolicy(TokenCredential credential, bool disableChallengeResourceVerification, bool enableContinuousEvaluation = false) : base(credential, Array.Empty<string>())
```

We generally avoid acronyms in .NET unless they're well-known and widely used.   Conventions also say "don't use consecutive capitals."

Reply:
I don't see any usage of this argument but we don't want it anyway. If the policy is capable of handling CAE challenges, it should request CAE tokens i.e. tell the authority it has that capability.

Reply:
@chlowell you're right, I forgot to remove this parameter when you told me that we won't be exposing a flag to the clients to enable CAE because it's going to be handled by the SDK by default.